### PR TITLE
[fem] Add linear corotated model

### PR DESCRIPTION
**Pull request by xuchenhan-tri**
This change is [Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/19170)

### CODE REVIEW

**Review by xuchenhan-tri — COMMENTED** [review]
-(status: do not review) +@amcastro-tri for feature review please. 

> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-:-NSrXJzu2Z7earR_CnTE:bq9kuvr)* status: LGTM missing from assignee amcastro-tri, needs platform reviewer assigned, needs at least two assigned reviewers (waiting on @amcastro-tri)

**Review by amcastro-tri — COMMENTED** [review]
> Reviewed 10 of 21 files at r1, 2 of 10 files at r2.
> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/19170)* status: 9 unresolved discussions, LGTM missing from assignee amcastro-tri, needs platform reviewer assigned, needs at least two assigned reviewers (waiting on @xuchenhan-tri)

___
*[`examples/multibody/deformable_torus/deformable_torus.cc` line 26 at r2](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTK1D9p4AsmMreKKt8l:-NTK1D9p4AsmMreKKt8m:b-mthnl9) ([raw file](https://github.com/RobotLocomotion/drake/blob/cb6b8f65f1e5d0c2e5ae6c260beb990beb429257/examples/multibody/deformable_torus/deformable_torus.cc#L26)):*
> ```cpp
> DEFINE_double(E, 1e4, "Young's modulus of the deformable body [Pa].");
> DEFINE_double(nu, 0.4, "Poisson's ratio of the deformable body, unitless.");
> DEFINE_double(density, 1e3, "Mass density of the deformable body [kg/m³].");
> ```

are these changes intentional?
___
*[`multibody/fem/acceleration_newmark_scheme.cc` line 35 at r2](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTK0m747Ixu0ZGfdz1q:-NTK0m747Ixu0ZGfdz1r:b-h5x299) ([raw file](https://github.com/RobotLocomotion/drake/blob/cb6b8f65f1e5d0c2e5ae6c260beb990beb429257/multibody/fem/acceleration_newmark_scheme.cc#L35)):*
> ```cpp
>   state->SetVelocities(vn + dt() * (gamma_ * a + (1.0 - gamma_) * an));
>   state->SetAccelerations(a);
>   state->SetTimeStepPositions(state->GetPositions());
> ```

shouldn't this happen *before* the `SetPositions()` above?
Also, wouldn't we like this to happen in `DiscreteTimeIntegrator::AdvanceOneTimeStep()` so that we do not need to do it for every new integrator?
___
*[`multibody/fem/deformable_body_config.h` line 14 at r2](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTK1U4ME3PrzdDCxYOc:-NTK1U4ME3PrzdDCxYOd:bjdn932) ([raw file](https://github.com/RobotLocomotion/drake/blob/cb6b8f65f1e5d0c2e5ae6c260beb990beb429257/multibody/fem/deformable_body_config.h#L14)):*
> ```objc
> /** Types of material models for the deformable body. */
> enum class MaterialModel {
>   /** Linear corotational model as described in [Han, 2023]. It provides a
> ```

nit, `[Han et al., 2023]`, as we do in other places.



_Code quote:_
```objc
[Han, 2023]
```
___
*[`multibody/fem/deformation_gradient_data.h` line 59 at r2](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTK29m1AeYIi8HNqVMW:-NTK29m1AeYIi8HNqVMX:b-oygeo4) ([raw file](https://github.com/RobotLocomotion/drake/blob/cb6b8f65f1e5d0c2e5ae6c260beb990beb429257/multibody/fem/deformation_gradient_data.h#L59)):*
> ```objc
>   void UpdateData(
>       std::array<Matrix3<T>, num_locations> deformation_gradient,
>       std::array<Matrix3<T>, num_locations> time_step_deformation_gradient) {
> ```

ditto with naming. I believe `previous_step_deformation_gradient` or similar naming consistent with the names you used for state would be more descriptive.
___
*[`multibody/fem/fem_state.h` line 68 at r2](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTK-EA7EzFfczjJCBI1:-NTK-EA7EzFfczjJCBI2:b-nhchde) ([raw file](https://github.com/RobotLocomotion/drake/blob/cb6b8f65f1e5d0c2e5ae6c260beb990beb429257/multibody/fem/fem_state.h#L68)):*
> ```objc
>    @{ */
>   const VectorX<T>& GetPositions() const;
>   const VectorX<T>& GetTimeStepPositions() const;
> ```

maybe `GetPreviousTimeStepPositions()`? or, to be consistent with `GetPositions()`, maybe `GetPreviousPositions()`? 
___
*[`multibody/fem/fem_state.cc` line 33 at r2](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTK-aHs5D8gunOkcvyi:-NTK-aHt7iRyfD7w11Gd:b-cypuoz) ([raw file](https://github.com/RobotLocomotion/drake/blob/cb6b8f65f1e5d0c2e5ae6c260beb990beb429257/multibody/fem/fem_state.cc#L33)):*
> ```cpp
> const VectorX<T>& FemState<T>::GetTimeStepPositions() const {
>   return get_context()
>       .get_discrete_state(system_->fem_time_step_position_index())
> ```

ditto, `fem_previous_time_step_position_index()` or `fem_previous_position_index()`
___
*[`multibody/fem/test/deformation_gradient_data_test.cc` line 83 at r2](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTK3CFkEjIa_6d98yFe:-NTK3CFkEjIa_6d98yFf:bhwro64) ([raw file](https://github.com/RobotLocomotion/drake/blob/cb6b8f65f1e5d0c2e5ae6c260beb990beb429257/multibody/fem/test/deformation_gradient_data_test.cc#L83)):*
> ```cpp
>   // clang-format on
>   const auto F0 = F;
>   dummy_data.UpdateData({F}, {F0});
> ```

Maybe worth trying two different values to ensure they are stored in two distinct memory locations?
___
*[`multibody/fem/test/deformation_gradient_data_test.cc` line 87 at r2](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTK38po-aPVmA8iAxkS:-NTK38po-aPVmA8iAxkT:b9pgfmw) ([raw file](https://github.com/RobotLocomotion/drake/blob/cb6b8f65f1e5d0c2e5ae6c260beb990beb429257/multibody/fem/test/deformation_gradient_data_test.cc#L87)):*
> ```cpp
>   EXPECT_TRUE(CompareMatrices(dummy_data.deformation_gradient()[0], F));
>   EXPECT_TRUE(
>       CompareMatrices(dummy_data.double_deformation_gradient()[0], 2.0 * F));
> ```

unit test `previous_deformation_gradient()` as well?
___
*[`multibody/fem/test/volumetric_element_test.cc` line 64 at r2](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTK5agV2AWqP0zY4kON:-NTK5agV2AWqP0zY4kOO:b-29w9r7) ([raw file](https://github.com/RobotLocomotion/drake/blob/cb6b8f65f1e5d0c2e5ae6c260beb990beb429257/multibody/fem/test/volumetric_element_test.cc#L64)):*
> ```cpp
>           element_data->resize(1);
>           const FemState<AD> fem_state(fem_state_system_.get(), &context);
>           (*element_data)[0] = (this->elements_)[0].ComputeData(fem_state);
> ```

maybe here you could set element_q0 to garbage just to stress test that it is not used here?

**Review by amcastro-tri — COMMENTED** [review]
per f2f. Xuchen will consider making q0 a cache entry rather than additional state.

> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-:-NTKizCwAGZGRasMshl0:ba0fp1i)* status: 9 unresolved discussions, LGTM missing from assignee amcastro-tri, needs platform reviewer assigned, needs at least two assigned reviewers (waiting on @xuchenhan-tri)

**Review by amcastro-tri — COMMENTED** [review]
> Reviewed 4 of 21 files at r1.
> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/19170)* status: 16 unresolved discussions, LGTM missing from assignee amcastro-tri, needs platform reviewer assigned, needs at least two assigned reviewers (waiting on @xuchenhan-tri)

___
*[`multibody/fem/linear_corotated_model.cc` line 37 at r2](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTKrVlMC1r3BD9RuTyF:-NTKrVlMC1r3BD9RuTyG:btyn8gm) ([raw file](https://github.com/RobotLocomotion/drake/blob/cb6b8f65f1e5d0c2e5ae6c260beb990beb429257/multibody/fem/linear_corotated_model.cc#L37)):*
> ```cpp
> /* First Piola stress P is the derivative of energy density ϕ with respect
>  to deformation gradient F. In Einstein notation, we have
>      εᵢⱼ = 0.5 RₖᵢFₖⱼ + 0.5 ₖᵢFₖᵢRₖⱼ - δᵢⱼ.
> ```

spurious left indexes. 
___
*[`multibody/fem/linear_corotated_model.cc` line 37 at r2](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTKr_f60ITz0F5AaAVd:-NTKr_f60ITz0F5AaAVe:b-dxn69m) ([raw file](https://github.com/RobotLocomotion/drake/blob/cb6b8f65f1e5d0c2e5ae6c260beb990beb429257/multibody/fem/linear_corotated_model.cc#L37)):*
> ```cpp
> /* First Piola stress P is the derivative of energy density ϕ with respect
>  to deformation gradient F. In Einstein notation, we have
>      εᵢⱼ = 0.5 RₖᵢFₖⱼ + 0.5 ₖᵢFₖᵢRₖⱼ - δᵢⱼ.
> ```

Notice this is the first time you define epsilon in this scope. You might want to write the expression for the energy density phi in terms of epsilon so that the derivative below is not a surprise. 
___
*[`multibody/fem/linear_corotated_model.cc` line 59 at r2](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTKs3r89P2azemG_0kq:-NTKs3r90_-3UonlrQKY:bn499q5) ([raw file](https://github.com/RobotLocomotion/drake/blob/cb6b8f65f1e5d0c2e5ae6c260beb990beb429257/multibody/fem/linear_corotated_model.cc#L59)):*
> ```cpp
> /*
>  Calculates ∂Pᵢⱼ/∂Fₐᵦ, where
>      Pᵢⱼ = 2μ * Rᵢₖ *εₖⱼ + λ * εₖₖ * Rᵢⱼ,
> ```

minor, additional space between `*` and `epsilon` for consistency
___
*[`multibody/fem/linear_corotated_model.cc` line 62 at r2](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTKsFIcBTWpQ32W2qsK:-NTKsFIcBTWpQ32W2qsL:bh1o5vk) ([raw file](https://github.com/RobotLocomotion/drake/blob/cb6b8f65f1e5d0c2e5ae6c260beb990beb429257/multibody/fem/linear_corotated_model.cc#L62)):*
> ```cpp
>  So,
>      ∂Pᵢⱼ/∂Fₐᵦ = 2μ * Rᵢₖ * ∂εₖⱼ/∂Fₐᵦ + λ * ∂εₖₖ/∂Fₐᵦ * Rᵢⱼ,
>  We use the calculation result from before that
> ```

nit, I'd replace "before" for something like: the calculation result documented in CalcFirstPiolaStressImpl.
In case this file ever gets updated and things move around.



_Code quote:_
```cpp
CalcFirstPiolaStressImpl
```
___
*[`multibody/fem/linear_corotated_model_data.h` line 42 at r2](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTKnF9X7rzgi4z2Lmpe:-NTKnF9X7rzgi4z2Lmpf:bd8q17n) ([raw file](https://github.com/RobotLocomotion/drake/blob/cb6b8f65f1e5d0c2e5ae6c260beb990beb429257/multibody/fem/linear_corotated_model_data.h#L42)):*
> ```objc
> 
>  private:
>   friend DeformationGradientData<LinearCorotatedModelData<T, num_locations>>;
> ```

could you document here why the friendship is needed?
___
*[`multibody/fem/linear_corotated_model_data.cc` line 24 at r2](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTKno8T6p5hibRZZajs:-NTKno8T6p5hibRZZajt:b-8lr80x) ([raw file](https://github.com/RobotLocomotion/drake/blob/cb6b8f65f1e5d0c2e5ae6c260beb990beb429257/multibody/fem/linear_corotated_model_data.cc#L24)):*
> ```cpp
>       this->time_step_deformation_gradient();
>   for (int i = 0; i < num_locations; ++i) {
>     Matrix3<T>& local_R0 = R0_[i];
> ```

nit, const here and bellow
___
*[`multibody/fem/linear_corotated_model_data.cc` line 28 at r2](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTKoQvpBw7_8BddVO02:-NTKoQvpBw7_8BddVO03:boz7t61) ([raw file](https://github.com/RobotLocomotion/drake/blob/cb6b8f65f1e5d0c2e5ae6c260beb990beb429257/multibody/fem/linear_corotated_model_data.cc#L28)):*
> ```cpp
>     Matrix3<T> unused_S;
>     internal::PolarDecompose<T>(F0[i], &local_R0, &unused_S);
>     Matrix3<T> R0tF = local_R0.transpose() * F[i];
> ```

minor, maybe call R0F this something like `corotated_strain`(transpose) or `S_corotated`(transpose)?

**Review by amcastro-tri — COMMENTED** [review]
I was confused also!

I gave this more thought and, at a system level, I don't think q0 is state anymore (sorry!).
Now, I do not think we need to put this on hold for this. We could go either way at this level. Yes, we are using a context under the hood for caching, but for as long as we do not declare q0 as additional state at the MultibodyPlant level, we could implement it whichever way is more convenient for us.

Ok, leh me explain. I simply used Sherm's criteria for determining what is a cache entry or what is state. According to that, a quantity is (I'd say **can be**) a cache entry if it can be computed given the state variables. I bolded **can be** because we can always make any cache entry "state", with the result that now our state is no longer the minimum set of variables that you'd need to represent it. I think q0 falls in this category.

For a discrete system we write `x[n+1] = f(x[n])` (without loss of generality, I am leaving time and actuation out).
Consider the contact forces for instance. With our SAP formulation, contact forces are a function of both x[n] (previous penetration) and x[n+1] (next time step velocity), lets write that `fc(x[n+1]; x[n])`. We do not store additional previous time step for this, were we wrong?
Well, it turns out that if you give me x[n], I can compute x[n+1] from `x[n+1] = f(x[n])` (an expensive computation of course, but I can always cache it, as we do today) and with that, now I can compute `fc(x[n+1]; x[n])`. Therefore I can always write a method to evaluate the contact forces simply as a function of `x[n]`. With our lagged stresses, the same thing happens. They are a function of both x[n+1] and x[n] but it turns out I can compute x[n+1] as `x[n+1] = f(x[n])` (again, expensive, but I can cache this).

TLDR; the bottom line is that at the MultibodyPlant level, making q0 additional state would probably be wrong (unless we do not care about having this "minimal set of state variables"). Here, we could go either way. I just wanted to provide a detailed view for us to understand that distinction. 

Thoughts?

> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-:-NTONiyF4gZoO0RbeV4T:b-sj9fet)* status: 13 unresolved discussions, LGTM missing from assignee amcastro-tri, needs platform reviewer assigned, needs at least two assigned reviewers (waiting on @xuchenhan-tri)

___
*[`examples/multibody/deformable_torus/deformable_torus.cc` line 26 at r2](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTK1D9p4AsmMreKKt8l:-NTONUhQCV7Rcjo9iRN1:b-juoucr) ([raw file](https://github.com/RobotLocomotion/drake/blob/cb6b8f65f1e5d0c2e5ae6c260beb990beb429257/examples/multibody/deformable_torus/deformable_torus.cc#L26)):*
<details><summary><i>Previously, xuchenhan-tri wrote…</i></summary><blockquote>

The original stiffness is set to the lower value to ensure the sim doesn't blow up with the big time steps we are taking. With the linear model I can afford to choose nice whole numbers again. I can revert these changes if they bother you.
</blockquote></details>

no, it doesn't bother me. I just wanted to make sure it wasn't a leftover from local experimentation.
It's great to see how much more stable this is!
Thanks for the explanation.

**Review by amcastro-tri — COMMENTED** [review]
Feature review done. :lgtm_strong: 

I'll wait for you to take a look at the discussion above before assigning platform in case you want to make changes. I am however satisfied with this as is.

> Reviewed 2 of 21 files at r1, 2 of 10 files at r2, 3 of 18 files at r3.
> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-:-NTOWt3o5Gzi-tGteEbn:bw2jbi5)* status: 7 unresolved discussions, needs platform reviewer assigned, needs at least two assigned reviewers (waiting on @xuchenhan-tri)

___
*[`multibody/fem/test/linear_corotated_model_data_test.cc` line 36 at r3](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTOVurf9o2XwLIpfwWk:-NTOVurf9o2XwLIpfwWl:b-niwfm0) ([raw file](https://github.com/RobotLocomotion/drake/blob/b5df009197b1d1c18154517687d3fe1a0ee486ec/multibody/fem/test/linear_corotated_model_data_test.cc#L36)):*
> ```cpp
>       CompareMatrices(linear_corotated_model_data.deformation_gradient()[0],
>                       Matrix3<double>::Identity()));
>   EXPECT_TRUE(CompareMatrices(
> ```

nit, couldn't these be EXPECT_EQ as well? (no  tolerance)
___
*[`multibody/fem/test/linear_corotated_model_data_test.cc` line 49 at r3](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTOW9ckAgyCBrBbNNtl:-NTOW9ckAgyCBrBbNNtm:br5gucs) ([raw file](https://github.com/RobotLocomotion/drake/blob/b5df009197b1d1c18154517687d3fe1a0ee486ec/multibody/fem/test/linear_corotated_model_data_test.cc#L49)):*
> ```cpp
>   LinearCorotatedModelData<double, kNumLocations> linear_corotated_model_data;
>   /* We set deformation gradient as F0 = R0*S0 where R0 is an arbitrary rotation
>    matrix and S0 is an arbitrary sysmmetric positive definite matrix. */
> ```

nit, typo



_Code quote:_
```cpp
sysmmetric
```
___
*[`multibody/fem/test/linear_corotated_model_data_test.cc` line 69 at r3](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTOWa842ADnudD2SXm4:-NTOWa842ADnudD2SXm5:b-z0716c) ([raw file](https://github.com/RobotLocomotion/drake/blob/b5df009197b1d1c18154517687d3fe1a0ee486ec/multibody/fem/test/linear_corotated_model_data_test.cc#L69)):*
> ```cpp
>   linear_corotated_model_data.UpdateData({F}, {F0});
>   EXPECT_TRUE(CompareMatrices(
>       linear_corotated_model_data.deformation_gradient()[0], F));
> ```

nit, these comparisons for F and F0 can be exact?
___
*[`multibody/fem/test/linear_corotated_model_data_test.cc` line 74 at r3](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTOWkhr2k5QV8TJjJnY:-NTOWkhr2k5QV8TJjJnZ:b-1b3uo0) ([raw file](https://github.com/RobotLocomotion/drake/blob/b5df009197b1d1c18154517687d3fe1a0ee486ec/multibody/fem/test/linear_corotated_model_data_test.cc#L74)):*
> ```cpp
>   EXPECT_TRUE(CompareMatrices(linear_corotated_model_data.R0()[0], R0, kTol));
>   const Matrix3<double> expected_strain =
>       0.5 * (R0.transpose() * F + F.transpose() * R0) -
> ```

btw, nice test!

**Review by EricCousineau-TRI — COMMENTED** [review]
:lgtm: platform, nice!

-(release notes: none) +(release notes: feature) due to introduction of `kLinearCorotated` in C++ and Python API

> Reviewed 4 of 21 files at r1, 4 of 10 files at r2, 17 of 18 files at r3, 2 of 2 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-:-NTU08f8BpwEFurUYRZr:b-b9g7xm)* status: 4 unresolved discussions (waiting on @amcastro-tri and @xuchenhan-tri)

___
*[`multibody/fem/linear_corotated_model.h` line 28 at r4](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTTy7adBrIy0Vc8oYV1:-NTTy7adBrIy0Vc8oYV2:b-7yqb5t) ([raw file](https://github.com/RobotLocomotion/drake/blob/aa9bf50469a0c9f61e9f028069cec13f007af9cc/multibody/fem/linear_corotated_model.h#L28)):*
> ```objc
>  when needed.
> 
>  [Han et al., 2023] Han, Xuchen, Joseph Masterjohn, and Alejandro Castro. "A
> ```

Woot!
___
*[`multibody/fem/linear_corotated_model.h` line 53 at r4](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTTyqFE01rr4OsDz9Ig:-NTTyqFE01rr4OsDz9Ih:bnw4ykq) ([raw file](https://github.com/RobotLocomotion/drake/blob/aa9bf50469a0c9f61e9f028069cec13f007af9cc/multibody/fem/linear_corotated_model.h#L53)):*
> ```objc
>   const T& poissons_ratio() const { return nu_; }
> 
>   /* Returns the shear modulus (Lame's second parameter) which is given by
> ```

nit (uber minor) Some places say `Lame`, others say `Lamé`.

Consider being consistent for ease of simple text-based searc?
Maybe just use `Lame`, but in one location, say something like "Lame (sp. Lamé)"

**Review by amcastro-tri — COMMENTED** [review]
Excellent work. Here is an additional :lgtm:  for you :-) 

> Reviewed 1 of 10 files at r2, 2 of 2 files at r4.
> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-:-NTURZx40KLwLm4gPaw1:b-oxk71w)* status: 1 unresolved discussion (waiting on @xuchenhan-tri)

___
*[`multibody/fem/linear_corotated_model_data.cc` line 28 at r2](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTKoQvpBw7_8BddVO02:-NTUR0G0D8ROIPI02tF9:b-rnhrnk) ([raw file](https://github.com/RobotLocomotion/drake/blob/cb6b8f65f1e5d0c2e5ae6c260beb990beb429257/multibody/fem/linear_corotated_model_data.cc#L28)):*
<details><summary><i>Previously, xuchenhan-tri wrote…</i></summary><blockquote>

I do think of it as a deformation gradient, but in a different (corotated) frame.
</blockquote></details>

fair, at leas the symbol is nicer to read.
___
*[`multibody/plant/test/deformable_integration_test.cc` line 211 at r3](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTOTdjhD8eTjxtr3T_D:-NTURMBR0D4j-Ei1kf57:bpqeol8) ([raw file](https://github.com/RobotLocomotion/drake/blob/b5df009197b1d1c18154517687d3fe1a0ee486ec/multibody/plant/test/deformable_integration_test.cc#L211)):*
<details><summary><i>Previously, xuchenhan-tri wrote…</i></summary><blockquote>

I used a larger time step and reduced the force threshold. But I have to slightly increase the velocity threshold to accommodate the larger slipping in stiction. I manually verified that decreasing sigma can help push down the velocity threshold again.
</blockquote></details>

nice! I am hopping the new formulation I'm writing will be superior in this regard, since we'll be able to provide a "stiction tolerance" (with units of velocity) directly.

**Review by xuchenhan-tri — COMMENTED** [review]
-(release notes: feature) +(release notes: none) as deformable is still considered as an experimental feature at the moment.

From release playbook:

> Some features under development (eg, deformables as of this writing) may have no-release-notes policies, as their APIs although public are not yet fully supported. Be sure to take note of which these are, or ask on #platform_review slack.

> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-:-NTUYGLa3iyYDhpMFKLn:bd5sxhh)* status: :shipit: complete! all discussions resolved, LGTM from assignees amcastro-tri,EricCousineau-TRI(platform) (waiting on @amcastro-tri and @EricCousineau-TRI)

___
*[`multibody/fem/linear_corotated_model.h` line 53 at r4](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTTyqFE01rr4OsDz9Ig:-NTUXnt4BNS2jZBPVCR4:b-gij2kf) ([raw file](https://github.com/RobotLocomotion/drake/blob/aa9bf50469a0c9f61e9f028069cec13f007af9cc/multibody/fem/linear_corotated_model.h#L53)):*
<details><summary><i>Previously, EricCousineau-TRI (Eric Cousineau) wrote…</i></summary><blockquote>

nit (uber minor) Some places say `Lame`, others say `Lamé`.

Consider being consistent for ease of simple text-based searc?
Maybe just use `Lame`, but in one location, say something like "Lame (sp. Lamé)"
</blockquote></details>

I have opted for `Lamé` all the way through (except for in code obviously).
___
*[`multibody/plant/test/deformable_integration_test.cc` line 211 at r3](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-NTOTdjhD8eTjxtr3T_D:-NTUY23a0mJWYUKP49Fb:b-83sjoh) ([raw file](https://github.com/RobotLocomotion/drake/blob/b5df009197b1d1c18154517687d3fe1a0ee486ec/multibody/plant/test/deformable_integration_test.cc#L211)):*
<details><summary><i>Previously, amcastro-tri (Alejandro Castro) wrote…</i></summary><blockquote>

nice! I am hopping the new formulation I'm writing will be superior in this regard, since we'll be able to provide a "stiction tolerance" (with units of velocity) directly.
</blockquote></details>

Nice!

**Review by EricCousineau-TRI — COMMENTED** [review]
Ooh, yup - sorry and thanks!

> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/19170#-:-NTUhE1B9IJJ9htN0g6V:b-woe7ai)* status: :shipit: complete! all discussions resolved, LGTM from assignees amcastro-tri,EricCousineau-TRI(platform) (waiting on @amcastro-tri)

**Review by amcastro-tri — COMMENTED** [review]
> Reviewed 6 of 6 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake/19170)* status: :shipit: complete! all discussions resolved, LGTM from assignees amcastro-tri,EricCousineau-TRI(platform) (waiting on @xuchenhan-tri)
